helm/chaos-mesh/templates/secrets-configuration.yaml

```diff
@@ -199,6 +229,11 @@ webhooks:
         path: /validate-auth
     failurePolicy: Fail
     name: vauth.kb.io
+    timeoutSeconds: {{ $timeoutSeconds }}
```

Comment:
```suggestion
    {{- if $supportTimeoutSeconds }}
    timeoutSeconds: {{ $timeoutSeconds }}
    {{- end }}
```

Reply:
sorry for my carelessness, should check `supportTimeoutSeconds` before using `timeoutSeconds`